doc/release-notes.rst

```diff
@@ -90,6 +94,30 @@ Major Changes from Kraken
     ``osd_snap_trim_sleep``, and ``osd_scrub_sleep`` have been
     reimplemented to work efficiently.  (These are used in some cases
     to work around issues throttling background work.)
+  * The crush ``choose_args`` encoding has been changed to make it
+    architecture-independent.
+  * Pools are now expected to be associated with the application using them.
+    Upon completing the upgrade to Luminous, the cluster will attempt to associate
+    existing pools to known applications (i.e. CephFS, RBD, and RGW). In-use pools
+    that are not associated to an application will generate a health warning. Any
+    unassociated pools can be manually associated using the new
+    ``ceph osd pool application enable`` command. For more details see
+    `associate pool to application <../rados/operations/pools/#associate-pool-to-application>`_
+    in the documentation.
+  * Added new commands ``pg force-recovery`` and ``pg-force-backfill``. Use them to
+    boost recovery or backfill priority of specified pgs, so they're
+    recovered/backfilled before any other. Note that these commands don't
+    interrupt ongoing recovery/backfill, but merely queue specified pgs before
+    others so they're recovered/backfilled as soon as possible. New commands ``pg
+    cancel-force-recovery`` and ``pg cancel-force-backfill`` restore default
+    recovery/backfill priority of previously forced pgs.
+
+- *ceph-mon*:
+
+  * Added new configuration "public bind addr" to support dynamic
+    environments like Kubernetes. When set the Ceph MON daemon could
+    bind locally to an IP address and advertise a different IP address
+    ``public addr`` on the network.
```

Comment:
This is mixing in the detailed compat notes into the feature highlights section.  This stuff goes in the lower section, 'Upgrade compatibility notes'

Reply:
Actually, let's move the 'pg force-*' bits to teh CLi changes, and the choose-args encoding to the compat notes.  Pool metadata can stay.

Reply:
ok